crates/release_plz/src/commands/init_github.rs

```diff
@@ -1,4 +1,36 @@
-mod gh;
+use std::path::{Path, PathBuf};
+
+use clap::builder::PathBufValueParser;
+
+use super::manifest::ManifestCommand;
+
+/// Initialize release-plz for the current GitHub repository.
+///
+/// Stores the necessary tokens in the GitHub repository secrets and generates the release-plz.yml
+/// GitHub Actions workflow file.
+#[derive(clap::Parser, Debug)]
+pub struct InitGithub {
```

Comment:
Renamed from Init to prepare a possible init command that just initializes the release-plz.yml file instead of the entire github repo. I could go either way about whether this is necessary. (No bug problem changing this back to init and perhaps adding sub fields that specify whether to initialize the repo secrets and actions.

Reply:
I prefer to have a single command for both and then have sub fields 👍